pytorch/serving/tests.yaml

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 ipex-serving-model-archive:
```

Comment:
perhaps we should make the test name with `cpu`.

Reply:
same with the below tests.

Reply:
added `cpu`